src/pages/Account/Account.scss

```diff
@@ -0,0 +1,252 @@
+$side-padding: 30px;
+$primary: #f8e71c;
+$black: #000000;
+$white: #fff;
+
+$dark: #1a1d24;
+$border-radius: 28px;
+$large-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.34);
+$login-max-width: 540px;
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+html {
+}
+
+body {
+  font: 1rem/1.3 "Rubik", "HelveticaNeue", "Helvetica Neue", Helvetica,
+    sans-serif;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  background-color: $black;
+  color: white;
+  padding: 0;
+}
```

Comment:
shouldn't we write that in `global.scss`?

Reply:
I copy pasted Mathieu's stylesheet. You're right we need to figure out what's global and what only applies to the account pages.